components/two-column-text-callout/Component.jsx

```diff
@@ -23,22 +24,30 @@ export default function TwoColumnTextCallout({
   showTopBorder = true,
   calloutsArray,
 }) {
+  const hasBothCallout = calloutsArray?.length === 2;
+
   return (
     <Container width="wide" marginTop={3} marginBottom={5}>
-      {showTopBorder && <hr className="su-border-black-30 su-rs-mb-3 su-h-1" />}
```

Comment:
I think we can leave this for now, but maybe this should be a bottom border like in the info section?